tests/api/validate.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { validateSchedules } from "@dataform/api";
+import { dataform } from "@dataform/protos";
+import { assert, config, expect } from "chai";
+
+describe("@dataform/api/validate", () => {
+  describe("validateSchedules", () => {
+
+    it("test_valid_schedule", () => {
```

Comment:
it's good practice to name jasmine testcases with human-readable strings. https://github.com/CareMessagePlatform/jasmine-styleguide#speak-human has some good guidance here.

for example, this one might be "returns no errors for valid schedules object"

Reply:
done